roles/loadbalancer/provision/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+- name: switch selinux to permissive
+  selinux: policy=targeted state=permissive
+
+#this is required to have the 22 port free on haproxy VIP's
+- name: setup ssh for loadbalancer
+  lineinfile: dest=/etc/ssh/sshd_config line="ListenAddress {{ installer.overcloud.loadbalancer.network.plain.ip | ipaddr('address') }}"
+
+- name: restart sshd service
+  service: name=sshd state=restarted
+
+- name: enable vlan kernel module
+  linefile:
+  create: yes
+  state: present
+  line: "exec /sbin/modprobe 8021q >/dev/null 2>&1"
+  mode: "0700"
+  owner: "root"
+  dest: /etc/sysconfig/modules/vlan.modules
+
+- name: create VLAN interface for plain network
+  template:
+  src: vlan-plain.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.oc_interface }}
+  when: installer.overcloud.loadbalancer.network.plain is defined
+
+- name: create VLAN interface for external network
+  template:
+  src: vlan-external.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.ex_interface }}
+  when: installer.overcloud.loadbalancer.network.external is defined
+
+- name: create VLAN interface for internal_api network
+  template:
+  src: vlan-internal_api.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.InternalApiNetworkVlanID }}
+  when: installer.overcloud.loadbalancer.network.internal_api is defined
+
+- name: create VLAN interface for storage network
+  template:
+  src: vlan-storage.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.StorageNetworkVlanID }}
+  when: installer.overcloud.loadbalancer.network.storage is defined
+
+- name: create VLAN interface for storage_mgmnt network
+  template:
+  src: vlan-storage_mgmt.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.StorageMgmtNetworkVlanID }}
+  when: installer.overcloud.loadbalancer.network.storage_mgmnt is defined
+
+- name: create VLAN interface for tenant network
+  template:
+  src: vlan-tenant.cfg.j2
+  dest: /etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.TenantNetworkVlanID }}
+  when: installer.overcloud.loadbalancer.network.tenant is defined
+
+- name: Install release tool on loadbancer
+  command: "yum localinstall -y {{ product.rpm }}"
+
+- name: Execute rhos-release for OSP-Director
+  command: rhos-release {{ installer.version.major}}
+
+- name: update all packages
+  yum:
+  name: *
+  state: latest
+
+- name: install haproxy service
+  yum:
+  name: haproxy
```

Comment:
well, I can keep writing "same" but you get the point :)
please go over all your playbooks and make sure they are indented :)